### ISSUE TITLE
JDStatusBarNotification fails to build on Xcode 15 beta

### ISSUE THREAD

**Issue by Dean151** [calimarkus/JDStatusBarNotification#125]
Somehow, trying to build JDStatusBarNotification on Xcode 15 beta provoke this error:
```
~/Library/Developer/Xcode/DerivedData/<redacted>/SourcePackages/checkouts/JDStatusBarNotification/JDStatusBarNotification/Public/JDStatusBarNotificationPresenter.m:12:9 'JDSBNotificationView.h' file not found
```

The file 'JDSBNotificationView.h' is indeed present in the private folder.
The library is attached to my project using Swift Package Manager.

Reproduction steps:
- Create a new iOS App project in Xcode 15 with default settings
- Add JDStatusBarNotification dependency using SPM, add it to the main target
- Build ; the error triggers

---

Not sure if this issue is a bug from Apple (Xcode is still in beta) ; or if it's linked with latest changes regarding Swift interoperability with C++ ; and/or libraries changes that have been made with Apple

**Comment by Dean151** [comment]
After investigation, the package refuse to build in standalone for `any iOS Device` when we remove the cSettings from the Package.swift file.
Might look like a Xcode bug. I'm going to fill a radar

**Comment by calimarkus** [comment]
I'm seeing the same, this is an import of a private header. Which should be working. Thanks for filing the radar!

**Comment by Dean151** [comment]
> I see that you are using .headerSearchPath("**") in your package manifest. Globs are not supported here, so you would have to explicitly specify the concrete paths you'd like to use here.
https://wwdc23.slack.com/archives/C0584F4S7FG/p1686246050406139?thread_ts=1686246050.204899&cid=C0584F4S7FG

Seems like they expect glob to stop working with glob.
Not specified anywhere in documentation tough
https://developer.apple.com/documentation/packagedescription/cxxsetting/headersearchpath(_:_:)

**Comment by Dean151** [comment]
Felt free to open a PR fixing the issue :)

**Comment by calimarkus** [comment]
Merged!

**Comment by calimarkus** [comment]
I published version 2.0.8 with it, thank you for investigating it and putting up a fix!

**Comment by Dean151** [comment]
My pleasure!
